src/main/resources/db/migration/V6__drop_pk_and_expiration_from_user_session.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE `user_session`
+    DROP PRIMARY KEY,
```

Comment:
What are you doing? In a production setting, this is unacceptable.

Reply:
The field that was indicated as primary key in `user_session` table was incorrect, it was the `token`. So I needed to drop it, in order to give the primary key to the `id`.
How should I have done this process differently?

Reply:
Looking at `V4__create_user_session_table.sql` the correct field is a primary key:
```
CREATE TABLE IF NOT EXISTS `user_session` (
  `id` BIGINT NOT NULL AUTO_INCREMENT,
  `token` VARCHAR(255) NOT NULL,
  `user_id` BIGINT NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE (`token`),
  FOREIGN KEY (`user_id`) REFERENCES `user`(`id`)
);
```

Reply:
<img width="502" alt="Screenshot 2024-07-01 at 9 52 27 AM" src="https://github.com/gdesiato/pure-synth/assets/10553375/4f96da55-f31b-4438-8c21-e9953da475d4">
This is the situation that I had in the database

Reply:
This is something you need to solve in your database. The migration files are not a way to do it. Migrations are supposed to allow for recreating the DB in a clean state. The way to solve it is one of:
1. Drop the database and recreate from scratch
2. Apply fixes manually and fix Flyway schema changes.

Considering your situation, I would do (1).

Reply:
I have one question. I drop the database, but do I do the migration based on the pre-existing migration files?

Reply:
The idea of Flyway migration is to create end state (same as production) database schema. It's to allow new people joining the project quickly setup database and track all changes to the DB schema. Please read more about Flyway migrations, to understand it if it's still not clear to you how it works. By applying migrations you will get to the final state of the DB. If it will still require some fixes, than you add new migrations on top of that. If you mess up your local DB, you need to bring it up to the state the codebase defines it however you like:
* recreate the DB from scratch
* fix it manually and fix checksums
* maybe there are some smart commands in Flyway, but I never used it